src/styles/blocks/stars--5.css

```diff
@@ -0,0 +1,44 @@
+.stars--5 {
+  display: flex;
+  margin-top: 4px;
+}
+
+.stars--5 .stars__star:nth-child(1) {
+  display: block;
+  background-image: url(../../images/star-active.svg);
+  width: 13px;
+  height: 12.2px;
+  margin-left: 22px;
```

Comment:
The `margin-left` for the first star creates an inconsistency with the rest of the stars. According to the checklist, be consistent with your margins. If there's no specific requirement for the first star to have a different margin, it should be the same as the others.